modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/ProcessBuilderSource.testcase

```diff
@@ -618,8 +618,6 @@ definition {
 	@description = "Verify that the time format in Details tab is following the system language"
 	@priority = "3"
 	test TimeFormatFollowsSystemLanguageInDetails {
-		property portal.upstream = "quarantine";
-
```

Comment:
Hey @sirfabios,

You can use other commits when removing the quarantined property. It will be easier to track the change commit.